src/middleware/withAuthorizedSpace.js

```diff
@@ -120,14 +122,22 @@ const authorize = async (space, ctx) => {
     capability: serve.transportHttp,
     authority: ctx.gatewayIdentity,
     principal: Verifier,
-    validateAuthorization: () => ok({})
+    validateAuthorization: () => ok({}),
+    // resolveDIDKey: async (did) => {
+    //   if (did === ctx.gatewayIdentity.did()) return ok(ctx.gatewayIdentity.toDIDKey())
+    //   throw new Error(`Unknown DID: ${did}`)
+    // }
```

Comment:
**suggestion:** Obviously, let's not have commented lines here, but…this makes me wonder if this is actually the correct way to do this after all, rather than a `Signer` with a swapped-in DID. The _correct_ correct way to do it is to actually resolve the DID document and get the key, but we don't currently publish DID documents for our `did:web:`s, so we're not there yet.

In any case, the thing to do right now is whatever's easiest, but maybe using `resolveDIDKey` makes sense? 🤔